feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature1 branch
```

Comment:
Its really good that u have decided to add the code out here

Reply:
Thanks for the comment amit

Reply:
well  done